### DISCUSSION TITLE
WPF Black Screen

### DISCUSSION THREAD

**Discussion by alexdatamaster** [cefsharp/CefSharp#4906] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

126.2.180

### Operating System

Windows 10

### Architecture

AnyCPU

### .Net Version

.Net 6.0

### Implementation

WPF

### Reproduction Steps

Load in Google Maps in browser.

### Expected behavior

The browser to load in correctly.

### Actual behavior

Browser comes in black.

### Does this problem also occur in the CEF Sample Application

Not Tested

**Comment by amaitland** [comment]
Did you test with https://github.com/cefsharp/CefSharp.MinimalExample?

I'm not seeing any issues in WPF on .Net 6 or 8 when loading Google maps, nor have there been any other reports of issues.

Did you check the log file for errors?

Did you add an app.manifest as per https://github.com/cefsharp/CefSharp/wiki/Quick-Start-For-MS-.Net-5.0-or-greater#40-add-appmanifest-to-your-application
